src/components/new-request/search.vue

```diff
@@ -312,20 +332,18 @@ export default class NewSearch extends Mixins(CommonMixin) {
   @Action setDoNameCheck!: ActionBindingIF
   @Action setEntityTypeCd!: ActionBindingIF
   @Action setExtendedRequestType!: ActionBindingIF
-  @Action setIsPersonsName!: ActionBindingIF
   @Action setJurisdiction!: ActionBindingIF
   @Action setLocation!: ActionBindingIF
   @Action setName!: ActionBindingIF
-  @Action setNameIsEnglish!: ActionBindingIF
   @Action setNoCorpNum!: ActionBindingIF
   @Action setPickEntityModalVisible!: ActionBindingIF
   @Action setRequestAction!: ActionBindingIF
   @Action startAnalyzeName!: ActionBindingIF
 
   // Local properties
   corpNumValid = true
-  locationDisabled = false
-  selectedCompanyType: CompanyType = null
+  request = null as RequestActionsI
+  selectedCompanyType = null as CompanyType
```

Comment:
This is a code style change only. I prefer to see a lot of "x = y" declarations.